akka-docs/src/main/paradox/typed/event-stream.md

```diff
@@ -0,0 +1,289 @@
+# EventStream
+
+You are viewing the documentation for the new actor APIs, to view the Akka Classic documentation, see @ref:[Classic Event Stream](../event-bus.md#event-stream).
+
+## Dependency
+
+<!--- #dependency --->
+The Akka dependencies are available from Akka's library repository. To access them there, you need to configure the URL for this repository.
+
+@@repository [sbt,Maven,Gradle] {
+id="akka-repository"
+name="Akka library repository"
+url="https://repo.akka.io/maven"
+}
+
+To use Akka Actor Typed, you must add the following dependency in your project:
+
+@@dependency[sbt,Maven,Gradle] {
+bomGroup=com.typesafe.akka bomArtifact=akka-bom_$scala.binary.version$ bomVersionSymbols=AkkaVersion
+symbol1=AkkaVersion
+value1="$akka.version$"
+group=com.typesafe.akka
+artifact=akka-actor-typed_$scala.binary.version$
+version=AkkaVersion
+}
+<!--- #dependency --->
+
+## Introduction
+
+The event stream is the main @ref:[Event Bus](#eventbus). of each actor system: it is used for
+carrying @ref:[log messages](logging.md) and @ref:[Dead Letters](#dead-letters) and may be
+used by the user code for other purposes as well. 
+
+It uses @ref:[Subchannel Classification](#subchannel-classification) which enables registering to related sets of channels.
+
+
+## How to use
+
+The following example demonstrates how a simple subscription works. Given a simple actor:
```

Comment:
We try to avoid "simple" and "just", better without them:
```suggestion
The following example demonstrates how a subscription works. Given an actor:
```